lib/exercism/homework.rb

```diff
@@ -19,11 +19,9 @@ def status(language)
     exercises = user.exercises.where(language: language)
 
     skipped = exercises.where.not(skipped_at: nil).pluck(:slug)
-    fetched = exercises.where.not(fetched_at: nil).pluck(:slug)
+    fetched = exercises.where('fetched_at IS NOT NUll AND iteration_count = ?', 0).pluck(:slug)
     recent = exercises.where.not(last_iteration_at: nil)
-             .order(:last_iteration_at).last
-
-    recent = Struct.new(:slug, :last_iteration_at).new("", "") if recent.nil?
+             .order(:last_iteration_at).last || struct_recent
```

Comment:
`Recent` hasn't been working out. We'll need to keep it since the CLI has a bug in this area, but let's not create a separate method for it. Just make it a long line.

I'd like to add the list of exercises that they've submitted something to. We can add that without breaking backwards compatibility on the client.